runelite-api/src/main/java/net/runelite/api/ItemComposition.java

```diff
@@ -51,6 +53,29 @@
 	 */
 	int getNote();
 
+	/**
+	 * Gets the model ID of the inventory item.
+	 *
+	 * @return the model ID
+	 */
+	int getInventoryModel();
+
+	/**
+	 * Gets the colors to replace with on the item.
```

Comment:
What does this mean? Without internal access this doesn't explain what this field is/does.